libs/src/services/ethContracts/governanceClient.js

```diff
@@ -0,0 +1,39 @@
+const ContractClient = require('../contracts/ContractClient')
+
+class GovernanceClient extends ContractClient {
```

Comment:
maybe lets add a comment here or trello card (or document somewhere) that we later need to implement rest of logic, since this PR adds only functionality required to unblock local dev